content/SCALE/GettingStarted/SCALEReleaseNotes.md

```diff
@@ -35,7 +39,26 @@ More details are available from [Nightly Upgrades]({{< relref "Upgrades.md" >}})
 
 {{< include file="/content/_includes/ReleaseScheduleWarning.md" >}}
 
-{{< releaselist name=scale-releases >}}<br>
+{{< releaselist name=scale-releases >}}
 {{< /expand >}}
 
+## Upgrade Notes
+
+* Users with unofficial apps installed should review app storage drivers to determine if any utilize the OpenEBS-ZFS container storage interface (CSI) before upgrading. This CSI is not supported in TrueNAS SCALE 24.04 ([Removal Notice](https://www.truenas.com/community/threads/openebs-zfs-driver-removal-notice.115026/)). Unofficial apps which use OpenEBS-ZFS CSI drivers should maintain functionality for existing deployments, but users are not able to make backups or restore any existing backup for those apps. New users are not able to install and deploy these apps.
+
+<!-- ### Upgrade Paths -->
+
+<!-- ## Component Versions -->
+
 ## Nightly Changelog
+
+Notable Changes:
+
+* OpenEBS-ZFS container storage interface (CSI) driver support is removed in TrueNAS SCALE 24.04 ([Removal Notice](https://www.truenas.com/community/threads/openebs-zfs-driver-removal-notice.115026/)).
+  
+  New and existing users who only use official apps are unaffected by this change, as these apps do not use OpenEBS-ZFS CSI drivers.
+  Unofficial apps are unaffected if they are configured as outlined below.
+  
+  Unofficial apps which use OpenEBS-ZFS CSI drivers should maintain functionality for existing deployments, but users are not able to make backups or restore any existing backup for those apps. New users are not able to install and deploy these apps.
+  
+  Maintainers of Unofficial Catalog apps using OpenEBS-ZFS CSI drivers should either begin to ship a CSI driver with the app or use the one provided in SCALE.
```

Comment:
I don't think we have a UI or formal catalog called Unofficial Catalog, so you can make this lower case.
```suggestion
  Maintainers of unofficial catalog apps using OpenEBS-ZFS CSI drivers should either begin to ship a CSI driver with the app or use the one provided in SCALE.
```